app/stac_api/models.py

```diff
@@ -153,13 +154,23 @@ class Link(models.Model):
     # added link_ to the fieldname, as "type" is reserved
     link_type = models.CharField(blank=True, null=True, max_length=150)
     title = models.CharField(blank=True, null=True, max_length=255)
+    hreflang = models.CharField(blank=True, null=True, max_length=32)
 
     class Meta:
         abstract = True
 
     def __str__(self):
         return f'{self.rel}: {self.href}'
 
+    def save(self, *args, **kwargs) -> None:
+        """Validate the hreflang"""
+        self.full_clean()
+
+        if self.hreflang is not None and self.hreflang != '' and not tags.check(self.hreflang):
+            raise ValidationError(_("Invalid value for hreflang"))
```

Comment:
Apparently the `language-tags` package [gives quite detailed error messages](https://github.com/OnroerendErfgoed/language-tags/blob/develop/language_tags/Tag.py#L347) (for example "The private-use subtag 'abcdefhijk' is too long."). Is it on purpose that these are not returned here? Looks good enough for me as it is, just to double check.

Reply:
Hmm no actually I haven't thought about that

Reply:
Included the error messages :+1: